src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Todo from "../Todo/Todo";
+import './TodoList.scss';
+import PropTypes from 'prop-types';
+
+function TodoList({todos}) {
+  return (
+    <ul className="todoList">
+        {todos.map(todo => (
+          <li className="todoInfo">
+            <Todo {...todo} key={todo.id}/>
+          </li>
```

Comment:
Key should be in the direct child component.
```suggestion
          <li className="todoInfo" key={todo.id}>
            <Todo {...todo} />
          </li>
```